frontend/src/components/eMIB/EmibIntroductionPage.jsx

```diff
@@ -11,32 +17,46 @@ const styles = {
 
 class EmibIntroductionPage extends Component {
   static propTypes = {
-    nextPage: PropTypes.func.isRequired
+    nextPage: PropTypes.func.isRequired,
+    // Provided by Redux
+    getTestMetaData: PropTypes.func
+  };
+
+  state = {
+    test_name_en: "",
+    test_name_fr: "",
+    markdown_en: "",
+    markdown_fr: ""
+  };
+
+  // loads the markdown content (english and french versions)
+  componentWillMount = () => {
+    this.props.getTestMetaData(TEST_DEFINITION.emib.sampleTest).then(response => {
+      // saving the background information markdown content in local states
```

Comment:
Can you just update the comment here with the right section name? Replace `background` with `overview`.
I'm also always forgetting to update these comments ;)

Reply:
Same, lol.
fixed in 43cf3d0